specification/dartLangSpec.tex

```diff
@@ -11829,7 +11829,40 @@ \subsection{Function Expressions}
 
 \rationale{%
 This definition guarantees that for any type $T$,
-\code{$T <:$ FutureOr<$\flatten{T}$>}.%
+\code{$T <:$ FutureOr<$\flatten{T}$>}. The proof is by induction on `T`:
```

Comment:
Maybe say "on the structure of `T`".
Not that there are any other obvious thing that the type is inductive in, but being explicit shouldn't hurt. 
(And type structure is inductive the way we've defined types).

Reply:
Done.